position/position.gno

```diff
@@ -165,24 +178,26 @@ func mint(params MintParams) (uint64, *u256.Uint, *u256.Uint, *u256.Uint) {
 	feeGrowthInside0LastX128 := u256.MustFromDecimal(_feeGrowthInside0LastX128.ToString())
 	feeGrowthInside1LastX128 := u256.MustFromDecimal(_feeGrowthInside1LastX128.ToString())
 
-	position := Position{
-		nonce:                    u256.Zero(),
-		operator:                 std.GetOrigCaller(),
-		poolKey:                  pl.GetPoolPath(params.token0, params.token1, params.fee),
-		tickLower:                params.tickLower,
-		tickUpper:                params.tickUpper,
-		liquidity:                liquidity,
-		feeGrowthInside0LastX128: feeGrowthInside0LastX128,
-		feeGrowthInside1LastX128: feeGrowthInside1LastX128,
-		tokensOwed0:              u256.Zero(),
-		tokensOwed1:              u256.Zero(),
-		burned:                   false,
-	}
+	position := newPosition(
+		u256.Zero(),
+		std.GetOrigCaller(),
+		pl.GetPoolPath(params.token0, params.token1, params.fee),
+		params.tickLower,
+		params.tickUpper,
+		liquidity,
+		feeGrowthInside0LastX128,
+		feeGrowthInside1LastX128,
+		u256.Zero(),
+		u256.Zero(),
+		false,
+	)
 	positions[tokenId] = position
 
 	return tokenId, liquidity, amount0, amount1
 }
 
+// -------------------------------------------------------------------------------------------- //
```

Comment:
what is purpose of this line ??

Reply:
Just for separate the function. forgot to remove it

Reply:
Just for separate public function regions. removed https://github.com/gnoswap-labs/gnoswap/pull/296/commits/bd068e18e93ce491b97de030db6467e790f495f4